app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_account_update_params, only: :update
+
+  # GET /resource/sign_up
+  def new
+    super
+  end
+
+  #POST /resource
+  def create
+    super do |resource|
+      if resource.valid?
+        @organization = Organization.create(name: params[:organization_name])      
+        resource.update(organization_id: @organization.id)
```

Comment:
avoid setting foreign key for an association, prefer setting association instance directly, e.g.: `resource.update(organization: @organization)`